containers/flatpak/cockpit-client.metainfo.xml.in

```diff
@@ -0,0 +1,48 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<component type="desktop-application">
+  <id>@FLATPAK_ID@</id>
+
+  <name>Cockpit Client</name>
+  <summary>Connect via ssh to servers running Cockpit</summary>
+
+  <metadata_license>CC0-1.0</metadata_license>
+  <project_license>LGPL-2.1-or-later</project_license>
+
+  <description>
+    <p>
```

Comment:
Do these tags need some special syntax to be translatable?

Reply:
Probably :rofl: